apps/teams-test-app/e2e-test-data/authentication.json

```diff
@@ -21,11 +21,50 @@
     {
       "title": "getUser API Call - Failure (Not full trust)",
       "version": ">2.25.0",
+      "hostSdkVersion": {
+        "web": "<3.1.0"
+      },
       "type": "callResponse",
       "platformsExcluded": ["iOS"],
       "boxSelector": "#box_getUser",
       "testUrlParams": [["appDefOverrides", "{\"isFullTrustApp\": false}"]],
       "expectedTestAppValue": "Error: Error: Error returned, code = 500, message = App does not have the required permissions for this operation"
+    },
+    {
+      "title": "getUser API Call - Failure (Not full trust and not Microsoft owned)",
+      "version": ">2.25.0",
+      "hostSdkVersion": {
+        "web": ">=3.1.0"
```

Comment:
This has to be ">=4.0.0" right? (Applies to all places)

Reply:
I think since the plan is to patch it back into the 3.x branch it's okay to keep it lower?

Reply:
Oh wait I'm not sure if you were commenting on the version number or the sign in front it it

I think it needs to be `>3.1.0` ? And then the test case above this should be `<=3.1.0`

Reply:
ah yes my bad, I didn't realize 3.1 was already out, will udate